app/models/manageiq/providers/azure/cloud_manager/provision/options_helper.rb

```diff
@@ -0,0 +1,9 @@
+module ManageIQ::Providers::Azure::CloudManager::Provision::OptionsHelper
+  def root_password
+    MiqPassword.decrypt(options[:root_password]) if options[:root_password]
+  end
+
+  def resource_group
+    ResourceGroup.find_by(:id => options[:resource_group]).name
```

Comment:
@bronaghs Would be better to have this return the object and have the caller decide what property to use.  You could also cache the result to be consistent with similar methods: [cloud_manager/provision/options_helper.rb#L19](https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/cloud_manager/provision/options_helper.rb#L19):

```ruby
@resource_group ||= ResourceGroup.find_by(:id => options[:resource_group])
```